jhub_apps/config_utils.py

```diff
@@ -58,3 +60,9 @@ class JAppsConfig(SingletonConfigurable):
         None,
         help="Disallow a set of frameworks to avoid spinning up apps using those frameworks",
     ).tag(config=True)
+
+    startup_apps = List(
+        trait=Any,  # TODO: Change this, use Instance() maybe or define a new type - https://traitlets.readthedocs.io/en/stable/defining_traits.html
```

Comment:
I just put this here for testing.  I want to somehow re-use the JHubAppConfig pydantic model ideally.

Reply:
We should accept git repos here alternatively